src/text/parsers/top_level.rs

```diff
@@ -136,26 +160,34 @@ mod parse_top_level_values_tests {
         );
     }
 
+    // Each of these input strings is followed by a ` 1` so the parser can definitively say the
+    // IVM is not followed by a `::`, which would make it an annotation instead.
     #[rstest]
-    #[case("$ion_1_0 ")]
-    #[case("   \r  \t \n $ion_1_0 ")]
-    #[case(" /*comment 1*/\n//comment 2\n   $ion_1_0 ")]
+    #[case("$ion_1_0 1")]
+    #[case("   \r  \t \n $ion_1_0 1")]
+    #[case(" /*comment 1*/\n//comment 2\n   $ion_1_0 1")]
     fn test_parse_ion_1_0_ivm(#[case] text: &str) {
         parse_test_ok(ion_version_marker, text, (1, 0));
     }
 
+    // Each of these input strings is followed by a ` 1` so the parser can definitively say the
+    // IVM is not followed by a `::`, which would make it an annotation instead.
     #[rstest]
-    #[case("$ion_1_1 ", (1, 1))]
-    #[case("/*hello!*/ $ion_1_1 ", (1, 1))]
-    #[case("$ion_2_0 ", (2, 0))]
-    #[case("\n \n $ion_2_0 ", (2, 0))]
-    #[case("$ion_5_8 ", (5, 8))]
-    #[case("$ion_21_99 ", (21, 99))]
+    #[case("$ion_1_1 1", (1, 1))]
+    #[case("/*hello!*/ $ion_1_1 1", (1, 1))]
+    #[case("$ion_2_0 1", (2, 0))]
+    #[case("\n \n $ion_2_0 1", (2, 0))]
+    #[case("$ion_5_8 1", (5, 8))]
+    #[case("$ion_21_99 1", (21, 99))]
```

Comment:
Quibble, but as I read these I wonder whether it would be clearer to use a symbol like `some_symbol` instead of the number `1` as more immediately self-documenting?

Reply:
I'd be ok with that, though the ` 1` is already being used in a lot of parser tests not visible in this diff. It'd need a broader change.